From 60ba8bd5b3575d0c7740571fbb4e681b21a49a82 Mon Sep 17 00:00:00 2001
From: Allan Sandfeld Jensen <allan.jensen@digia.com>
Date: Fri, 16 Aug 2013 18:27:07 +0200
Subject: [PATCH] ANGLE doesn't build with bison 3.0

https://bugs.webkit.org/show_bug.cgi?id=119798

Reviewed by Antti Koivisto.

Make glslang.y compatible with bison 3.0
by using %lex-param to set YYLEX_PARAM.

* src/compiler/glslang.y:

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@154109 268f45cc-cd09-0410-ab3c-d52691b4dbfc

Task-number: QTBUG-32913
Change-Id: I15505d31f0588c4d558b73befdb9d2358e29c1a3
Reviewed-by: Jocelyn Turcotte <jocelyn.turcotte@digia.com>
---
 Source/ThirdParty/ANGLE/src/compiler/glslang.y | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/ThirdParty/ANGLE/src/compiler/glslang.y b/Source/ThirdParty/ANGLE/src/compiler/glslang.y
index 3cad335..b41e95a 100644
--- a/Source/ThirdParty/ANGLE/src/compiler/glslang.y
+++ b/Source/ThirdParty/ANGLE/src/compiler/glslang.y
@@ -47,6 +47,7 @@ WHICH GENERATES THE GLSL ES PARSER (glslang_tab.cpp AND glslang_tab.h).
 %expect 1 /* One shift reduce conflict because of if | else */
 %pure-parser
 %parse-param {TParseContext* context}
+%lex-param {YYLEX_PARAM}
 
 %union {
     struct {
-- 
1.8.4.2

